nibabel/fileslice.py

```diff
@@ -767,9 +767,12 @@ def strided_scalar(shape, scalar=0.):
     strided_arr : array
         Array of shape `shape` for which all values == `scalar`, built by
         setting all strides of `strided_arr` to 0, so the scalar is broadcast
-        out to the full array `shape`.
+        out to the full array `shape`. `strided_arr` is flagged as not
+        `writeable`.
```

Comment:
What kind of comment are you thinking? Noting the numpy bug (with some level of detail, or reference to this PR) or the reasoning that setting one element of the array would set all (and thus is probably not the intended behavior)?